recipe.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Spaghetti Carbonara",
+  "ingredients": [
+    "200g pancetta or thick cut bacon, diced",
+    "4 yolks",
+    "400g spaghetti",
+    "1 cup grated parmesan",
+    "1 Tbsp olive oil",
+    "2 garlic cloves, minced",
+    "Salt and black pepper to taste"
+  ],
+  "instructions": "1 Heat pasta water: Put a large pot of salted water on to boil (1 Tbsp salt for every 2 quarts of water.) 2 Sauté pancetta/bacon and garlic: While the water is coming to a boil, heat the olive oil in a large sauté pan over medium heat. Add the bacon or pancetta and cook slowly until crispy. Add the garlic and cook another minute, then turn off the heat and put the pancetta and garlic into a large bowl. 3 Beat eggs and half of the cheese: In a small bowl, beat the yolks and mix in about half of the cheese. 4 Cook pasta: Once the water has reached a rolling boil, add the dry pasta, and cook, uncovered, at a rolling boil. 5 Toss pasta with pancetta/bacon: When the pasta is al dente (still a little firm, not mushy), use tongs to move it to the bowl with the bacon and garlic. Let it be dripping wet. Reserve some of the pasta water. Move the pasta from the pot to the bowl quickly, as you want the pasta to be hot. It's the heat of the pasta that will heat the eggs sufficiently to create a creamy sauce. Toss everything to combine, allowing the pasta to cool just enough so that it doesn't make the eggs curdle when you mix them in. (That's the tricky part.) 6 Add the beaten yolk mixture: Add the beaten yolks with cheese and toss quickly to combine once more. Add salt to taste. Add some pasta water back to the pasta to keep it from drying out. Serve at once with the rest of the parmesan and freshly ground black pepper.",
+  "total_calories": "2600",
```

Comment:
What data type is the value supposed to be here?

Reply:
It should be number, not a string, fixed!